### PR TITLE
Bahrain: Updated geometries and properties

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1512.

This PR includes:
- Updated record(s) for Manana (no localities to move to the neighbourhood placetype)
- Updates the country shape with a new, modified [US State Dept shape](https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/us-dshiu.json) shape.
- Updates the region shapes with new, modified [US State Dept shape](https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/us-dshiu.json) shapes.
- Validated geometries/GeoJSON

Once approved, I'll PIP and merge.

To do:
- [x] Update country shape
- [x] Update region shapes
- [x] Update locality of Manamah
- [x] Validate PR
- [x] PIP

### CODE REVIEW

File: data/856/690/49/85669049.geojson [hunk 1]
```diff
@@ -2,21 +2,20 @@
   "id": 85669049,
   "type": "Feature",
   "properties": {
-    "edtf:cessation":"uuuu",
+    "edtf:cessation":"2014",
```
Comment (nvkelso) [inline data/856/690/49/85669049.geojson]: Our convention has been to say `2014-01-01`?

Comment (nvkelso) [reply data/856/690/49/85669049.geojson]: Do we need a deprecation date on this, too, related to when we're superseding the record?

Comment (stepps00) [reply data/856/690/49/85669049.geojson]: :+1: I'll add a `edtf:superseded` property with today's date and figure out a more precise date of this feature's cessation.

**Comment by stepps00** [comment]
Took care of date comment and PIP work in 6ec4234, merging.

**Pull request merged**